notebooks/nyc_taxi_years_correlation.ipynb

```diff
@@ -181,8 +192,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 11.2 s, sys: 6.27 s, total: 17.5 s\n",
-      "Wall time: 17.7 s\n"
+      "CPU times: user 22.6 s, sys: 14.6 s, total: 37.2 s\n",
+      "Wall time: 37.2 s\n"
```

Comment:
This is the slowdown I'm talking about. More than 2x slower.

Reply:
oh thanks!  I hadn't noticed that at all!